src/plugins/apollo-plugin.js

```diff
@@ -41,7 +41,10 @@ export default Vue => {
 									...(isContentfulQuery(query) && isContentfulPreview && { preview: true })
 								}
 							});
-							result.call(this, { data });
+
+							if (data !== null) {
+								result.call(this, { data });
```

Comment:
Do we not want to send an empty result? What happens if the query doesn't send back a result in this case?

Reply:
I got and issue with data null at the moment I was working on the first approach, will take a new look about this.

Reply:
Thanks for investigating. If possible, find a use case (or force a user case) that results in data null and see what happens with the query. For example, does it timeout and fail if this result never gets called?